sdk/storage/Azure.Storage.Blobs/tests/BlobBaseClientTests.cs

```diff
@@ -71,6 +72,16 @@ public void Ctor_Uri()
             Assert.AreEqual(accountName, builder.AccountName);
         }
 
+        [Test]
+        public void BlobDownloadInfo_Dispose()
+        {
+            MockStream stream = new MockStream();
```

Comment:
Writing you own stream is totally fine but it can also be done easily with Moq:

```
var mock = new Mock<Stream>();

// use mock.Object

mock.Verify(foo => foo.Dispose(), Times.Once());
```

Reply:
This is on me - I suggested Josh not add a new dependency.  I'm fine doing it either way.

Reply:
Thanks I will keep this in mind for next time.